FirebasePerformance/Sources/Configurations/FPRRemoteConfigFlags.m

```diff
@@ -69,6 +75,8 @@ - (instancetype)initWithRemoteConfig:(FIRRemoteConfig *)config {
     _fprRemoteConfig = config;
     _userDefaults = [FPRConfigurations sharedInstance].userDefaults;
     self.fetchInProgress = NO;
+    self.appStartConfigFetchDelayInSeconds =
+        kFPRMinAppStartConfigFetchDelayInSeconds + arc4random_uniform(30);
```

Comment:
How is it know that 30 seconds is enough of a spread to address the issue?

Reply:
We are taking cue-s from the RC team to see what works well for them.